proto/insonmnia.go

```diff
@@ -0,0 +1,173 @@
+package sonm
+
+import (
+	"errors"
+	"fmt"
+	"math/big"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/c2h5oh/datasize"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/params"
+)
+
+var rateSuffixes = map[string]uint64{
+	"bit/s":  1,
```

Comment:
Placing `/s` everywhere is redundant. Just keep the map as `byteSizeSuffix = "bit": 1, ..." etc.

Reply:
this will require additional parsing

Reply:
Do we really need this? Do we want to specify data rate with smth like "10 Mb"?

Reply:
We don't strictly need this, but it seems more clean, because of deduplication. The parsing stage shouldn't get overcomplicated.